srcpkgs/libssc/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'libssc'
+pkgname=libssc
+version=0.2.2
+revision=1
+build_style=meson
+hostmakedepends="pkg-config protobuf protobuf-c"
+makedepends="glib-devel libqrtr-glib-devel protobuf-c-devel libqmi-devel python-devel"
+short_desc="Library to expose Qualcomm Sensor Core sensors"
+maintainer="Arseniy <me@adomerle.pw>"
+license="GPL-3.0-or-later"
+homepage="https://libssc.dylanvanassche.be/"
+distfiles="https://codeberg.org/DylanVanAssche/${pkgname}/archive/v${version}.tar.gz"
```

Comment:
remove pkg `pkgname` variable from distfiles, its not really variable, is harder to copy/paste and is actually longer.